test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -218,13 +224,17 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	for (auto i = 0; i < size; i++)
 	{
 		uint8_t opcode = weightsDefined ? randOpCodeWeight() : randOpCodeGen();
-		dev::eth::InstructionInfo info = dev::eth::instructionInfo((dev::eth::Instruction) opcode);
+		eth::Instruction inst = (eth::Instruction) opcode;
+		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos)
+		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
 		{
-			//Byte code is yet not implemented
-			if (_options.useUndefinedOpCodes == false)
+			if (_options.useUndefinedOpCodes)
+				code += toCompactHex(opcode, HexPrefix::DontAdd, 1);
```

Comment:
We do at least for the ones in `invalidOpcodes`, `c_instructionInfo` table has info on how many arguments are needed.

There's no sense in adding these opcodes into the code without arguments, if EVM supports them, it probably will try to interpret current stack as arguments